common/src/chain/transaction/signature/tests/utils.rs

```diff
@@ -142,24 +157,36 @@ pub fn generate_and_sign_tx(
     chain_config: &ChainConfig,
     rng: &mut (impl Rng + CryptoRng),
     destination: &Destination,
-    inputs_utxos: &[&TxOutput],
+    inputs_utxos: &[Option<TxOutput>],
```

Comment:
Why not `&[Option<&TxOutput>]`?

EDIT: You know what, forget it, this is test utils...